Sources/tart/OCI/Reference/Reference.g4

```diff
@@ -0,0 +1,13 @@
+grammar Reference;
```

Comment:
What the origin of this grammar? You compiled it from a spec? Would be nice to have a link here.

Reply:
No, it's a port of the old grammar that Tart used, but more (1) readable/editable and (2) strict w.r.t. potential path traversal (see d8a71ec).